src/components/AnonymousChat/child/ChatItemTargetText.tsx

```diff
@@ -44,7 +52,7 @@ const styles = StyleSheet.create({
     width: 24,
     height: 24,
     borderRadius: 12,
-    marginRight: 4
+    marginRight: AVATAR_MARGIN
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The margin value has been updated. Make sure this change doesn't affect the layout in an undesired way.
```diff
-    marginRight: 4
+    marginRight: AVATAR_MARGIN
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->